Model/Import/Category.php

```diff
@@ -954,13 +957,13 @@ private function saveCategories(): self
 
                     $time = !empty($rowData[CategoryModel::KEY_CREATED_AT])
                         ? strtotime($rowData[CategoryModel::KEY_CREATED_AT])
-                        : null;
+                        : 'now';
 
                     // entity table data
                     $entityRow = [
                         CategoryInterface::KEY_PARENT_ID => $parentCategory['entity_id'],
                         CategoryInterface::KEY_LEVEL => $parentCategory[CategoryInterface::KEY_LEVEL] + 1,
-                        CategoryInterface::KEY_CREATED_AT => (new DateTime($time))
+                        CategoryInterface::KEY_CREATED_AT => (new \DateTime($time))
```

Comment:
To match the general code style, let's import this and also remove the wrong use statement `Magento\Framework\Stdlib\DateTime`.

Reply:
We're actually using both, \DateTime to initialise from a string and Magento\Framework\Stdlib\DateTime for DATETIME_PHP_FORMAT